lib/srv/db/server.go

```diff
@@ -609,15 +621,27 @@ func (s *Server) ForceHeartbeat() error {
 // upgrades it to TLS, extracts identity information from it, performs
 // authorization and dispatches to the appropriate database engine.
 func (s *Server) HandleConnection(conn net.Conn) {
-	log := s.log.WithField("addr", conn.RemoteAddr())
+	remoteAddr := conn.RemoteAddr().String()
+	log := s.log.WithField("addr", remoteAddr)
```

Comment:
Nit: (`addr` is a bit too generic)
```suggestion
	log := s.log.WithField("client-ip", remoteAddr)
```